cleanup: move DomAdapter from angular2/core into angular2/platform/co…

vsavkin opened the pull request:
…mmon_dom

BREAKING CHANGE

Before

import {DomAdapter} from 'angular2/core';

After

import {DomAdapter} from 'angular2/platform/common_dom';

mary-poppins commented:
Merging PR #5370 on behalf of @alxhub to branch [presubmit-alxhub-pr-5370](https://github.com/angular/angular/tree/presubmit-alxhub-pr-5370).

mary-poppins commented:
Merging PR #5370 on behalf of @alxhub to branch [presubmit-alxhub-pr-5370](https://github.com/angular/angular/tree/presubmit-alxhub-pr-5370).

alxhub commented:
Seems like Travis is unhappy with the merge resolution. Just lmk when it's ready to merge again.

vsavkin committed:
cleanup: move DomAdapter from angular2/core into angular2/platform/common_dom

BREAKING CHANGE

Before

import {DomAdapter} from 'angular2/core';

After

import {DomAdapter} from 'angular2/platform/common_dom';

mary-poppins commented:
Merging PR #5370 on behalf of @alxhub to branch [presubmit-alxhub-pr-5370](https://github.com/angular/angular/tree/presubmit-alxhub-pr-5370).
